pkg/icingadb/types/comment_type.go

```diff
@@ -30,25 +29,7 @@ func (ct *CommentType) UnmarshalJSON(data []byte) error {
 
 // UnmarshalText implements the encoding.TextUnmarshaler interface.
 func (ct *CommentType) UnmarshalText(text []byte) error {
-	s := string(text)
-
-	i, err := strconv.ParseUint(s, 10, 64)
-	if err != nil {
-		return types.CantParseUint64(err, s)
-	}
-
-	c := CommentType(i)
-	if uint64(c) != i {
-		// Truncated due to above cast, obviously too high
-		return badCommentType(s)
-	}
```

Comment:
This check is missing in `CommentType.UnmarshalJSON`.

Reply:
But that method already uses uint8 which implies this check.

Reply:
Fair enough.